src/wwwroot/app/services/NoteService.js

```diff
@@ -12,12 +16,17 @@ export default class NoteService {
    }
 
    save(entity) {
+      this.convertDateToServerFormat(entity);
       if (entity.id) {
-         return _HttpService.put(`${NOTE_URL}/${entity.id}`, entity);
+         return _HttpService.put(`${NOTE_URL}/${entity.id}`, entity).then((comment) => {
+            return this.convertDateFromServerFormat(comment);
+         });
       } else {
          let currentUser = _UserService.getCurrentUser();
          entity.userId = currentUser.id;
-         return _HttpService.post(NOTE_URL, entity);
+         return _HttpService.post(NOTE_URL, entity).then((comment) => {
```

Comment:
you can call method in 'then' instead of creating a new clause. like 
**_HttpService.post(...).then(this.convertDateFromServerFormat)**